widgetssdk/src/main/java/com/glia/widgets/operator/OperatorRequestHandlerContract.kt

```diff
@@ -0,0 +1,24 @@
+package com.glia.widgets.operator
+
+import android.app.Activity
+import com.glia.androidsdk.Engagement
+import com.glia.androidsdk.comms.MediaUpgradeOffer
+import com.glia.widgets.engagement.domain.MediaUpgradeOfferData
+import com.glia.widgets.helper.OneTimeEvent
+import io.reactivex.Flowable
+
+internal interface OperatorRequestHandlerContract {
+    sealed interface State {
+        data class RequestMediaUpgrade(val data: MediaUpgradeOfferData) : State
+        data class OpenCallActivity(val mediaType: Engagement.MediaType) : State
+        object DismissAlertDialog : State
+
+        //add request permissions states
```

Comment:
Is it a TODO?

Reply:
Yes

Reply:
Let's add it as a TODO instead of a comment then. 😉